frotz.py

```diff
@@ -112,10 +117,10 @@ def __init__(self, handle):
         default_game_file = os.path.join(activity.get_bundle_path(), "Advent.z5")
         # when we return to the idle state, launch the default game
         # if read_file is called, that will override this
-        gobject.idle_add(self.start_game, default_game_file)
+        GLib.idle_add(self.start_game, default_game_file)
     
-    def _quit_cb(self, foo=None):
-        print "Quitting..."
+    def _quit_cb(self, vte, foo=None):
+        logging.debug("Quitting...")
         sys.exit(0)
```

Comment:
Use Activity.close instead?  See what Terminal does; when last tab is clsoed it calls `self.close()`.

Reply:
Done in eff0b61.